### PR TITLE
VPN-6488: update string for iOS exclude LAN feature

### PR DESCRIPTION
**Pull request by mcleinman**
## Description

Updated strings per ticket and conversation with Santiago.

## Reference

VPN-6488

## Checklist
    
- [x] My code follows the style guidelines for this project
- [x] I have not added any packages that contain high risk or unknown licenses (GPL,  LGPL, MPL, etc. consult with DevOps if in question)
- [x] I have performed a self review of my own code
- [x] I have commented my code PARTICULARLY in hard to understand areas
- [x] I have added thorough tests where needed

### CODE REVIEW

File: src/translations/strings.yaml [hunk 1]
```diff
@@ -354,9 +354,9 @@ settings:
     comment: Button label to clear all apps from the exclusions list
 
 localNetworkAccess:
-  label: Local network access
+  labelTitle: Exclude local network traffic
   subLabel: Access Apple CarPlay, AirDrop, AirPlay and local network devices like printers.
-  vpnMustBeOff: VPN must be off to edit “Local network access”
+  vpnMustBeOffTip: VPN must be off to edit “Exclude local network traffic”
```
Comment (flodolo) [inline src/translations/strings.yaml:359]: Wouldn't we better off with something more generic, e.g. `VPN must be off to edit this setting`?

**Review by flodolo — APPROVED** [review]

**Review by strseb — APPROVED** [review]

File: src/translations/strings.yaml (hunk 1 above)
Comment (mcleinman) [reply src/translations/strings.yaml:359]: Good suggestion. I'll defer to Santiago for this one, as he crafted the language here. He's off this week, but I'll ping him next week - and will hold this PR until he's back.

Comment (mcleinman) [reply src/translations/strings.yaml]: Santiago wanted to remove this warning completely, and instead put the text in the label. 

@flodolo  Please let me know if the preferred way to have this translated is two separate strings that we sometimes combine, or the way that it currently is written.

That is, should the 2 strings be:
```
subLabel: Access Apple CarPlay, AirDrop, AirPlay and local network devices like printers.
disabledSubLabel: Access Apple CarPlay, AirDrop, AirPlay and local network devices like printers. The VPN must be off to edit this setting.
```
(and we show either `subLabel` or `disabledSubLabel`)
or
```
subLabel: Access Apple CarPlay, AirDrop, AirPlay and local network devices like printers.
disabledRequirement: The VPN must be off to edit this setting.
```
(and we either show `subLabel` or `subLabel + disabledRequirement`)

Thanks!

Comment (flodolo) [reply src/translations/strings.yaml]: Combining would only work if you have the 2 strings clearly separated. In this case, you would need to know how to glue together 2 pieces based on the language.

**Review by flodolo — APPROVED** [review]

File: src/translations/strings.yaml (hunk 1 above)
Comment (mcleinman) [reply src/translations/strings.yaml]: That was my gut, so seems like we're good with how it's written. Thank you!

**Pull request merged**